search/engine.go

```diff
@@ -395,6 +395,17 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 		}
 	}()
 
+	// Search tweets in parallel to native search.
+	tweetsByLangChannel := make(chan map[string][]*elastic.SearchResult)
+	go func() {
+		if tweetsByLang, err := e.SearchTweets(query, sortBy, from, size, preference); err != nil {
+			log.Errorf("ESEngine.DoSearch - Error searching tweets: %+v", err)
+			tweetsByLangChannel <- map[string][]*elastic.SearchResult{}
```

Comment:
Should be map to pointer, not map to slice of pointers, right?

Reply:
fixed